assert/assertions_test.go

```diff
@@ -990,30 +990,44 @@ func TestContainsNotContains(t *testing.T) {
 	type A struct {
 		Name, Value string
 	}
-	list := []string{"Foo", "Bar"}
+	list := []string{"Hello", "World"}
+
+	byteSliceList := [][]byte{
+		[]byte("Hello"), []byte("World"),
+	}
 
 	complexList := []*A{
 		{"b", "c"},
 		{"d", "e"},
 		{"g", "h"},
 		{"j", "k"},
 	}
-	simpleMap := map[interface{}]interface{}{"Foo": "Bar"}
+
+	interfaceList := []interface{}{"Hello World", 5, 2, []byte("Goodbye")}
+
+	simpleMap := map[interface{}]interface{}{"Hello": "World"}
 	var zeroMap map[interface{}]interface{}
 
 	cases := []struct {
 		expected interface{}
 		actual   interface{}
 		result   bool
 	}{
-		{"Hello World", "Hello", true},
-		{"Hello World", "Salut", false},
-		{list, "Bar", true},
-		{list, "Salut", false},
+		{"Hello World", "World", true},
+		{"Hello World", "Earth", false},
+		{[]byte("Hello World"), []byte("World"), true},
+		{[]byte("Hello World"), []byte("Earth"), false},
+		{list, "World", true},
+		{list, "Earth", false},
+		{byteSliceList, []byte("World"), true},
+		{byteSliceList, []byte("Earth"), false},
```

Comment:
Ah, while reading through @dolmen's review I noticed that these examples from the doc aren't strictly in the unit tests because simpleMap is a `map[any]any` and not a `map[string]string`.
```go
//	assert.Contains(t, map[string]string{"Hello": "World"}, "Hello")
//	assert.NotContains(t, map[string]string{"Hello": "World"}, "Earth")
```
Could you add another couple of tests for those exact cases please.